packages/endpoint/0.5.0/dataset/events/fields/base-fields.yml

```diff
@@ -0,0 +1,33 @@
+- name: stream.type
```

Comment:
These fields can be removed now. We don't need the `stream` ones anymore.

Reply:
Thanks @ruflin yeah I'll be removing the entire `events` dataset directory shortly, this PR: https://github.com/elastic/kibana/pull/69589 merged first though

Reply:
Okay, I will address this in the next PR